src/Sign.Core/xlf/Resources.fr.xlf

```diff
@@ -159,8 +159,8 @@
       </trans-unit>
       <trans-unit id="VsixSignatureProviderSigning">
         <source>Signing OpenVsixSignTool job with {count} files.</source>
-        <target state="translated">Signature du travail OpenVsixSignTool avec {count} fichiers.</target>
-        <note>{count} is the number of files to be signed.  The literal text inside the brackets should not be localized.</note>
+        <target state="needs-review-translation">Signature du travail OpenVsixSignTool avec {count} fichiers.</target>
+        <note>{Placeholder="{count}"} is the number of files to be signed.</note>
```

Comment:
Should we keep the `The literal text inside the brackets should not be localized.` part here?

Reply:
Nope, it's not necessary.

Reply:
(`{Placeholder="..."}` already tells the loc engine that the text inside the brackets should not be localized.)